src/NuGet/Microsoft.CodeAnalysis.Workspaces.Common.nuspec

```diff
@@ -27,12 +27,8 @@ Supported Platforms:
     <serviceable>true</serviceable>
   </metadata>
```

Comment:
Does the dependencies list need to be updated as well? There's a bunch of other packages we're using too in src\Workspaces\Core\Portable\project.json:

```
    "System.Diagnostics.Contracts": "4.0.1",
    "System.Linq.Parallel": "4.0.1",
    "System.ObjectModel": "4.0.10",
    "System.Text.RegularExpressions": "4.1.0",
    "System.Threading.Tasks.Parallel": "4.0.1"
```